src/time_steppers.jl

```diff
@@ -152,29 +154,29 @@ function calculate_interior_source_terms!(grid::Grid, constants, eos, cfg, u, v,
                 @inbounds Gu[i, j, k] = (-u∇u(grid, u, v, w, i, j, k)
                                             + Gu_cori(grid, v, fCor, i, j, k)
                                             - δx_c2f(grid, pHY′, i, j, k) / (Δx * ρ₀)
-                                            + 𝜈∇²u(grid, u, 𝜈h, 𝜈v, i, j, k)
+                                            + ∂ⱼ_2ν_Σ₁ⱼ(i, j, k, grid, closure, eos, grav, u, v, w, T, S)
```

Comment:
Dumb question but is `Σ` the strain-rate tensor?

Reply:
quite! I will add some comments to `turbulence_closures.jl`